docs/_docs/02_features/custom-log-format.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: collection-browser-doc
+title: Custom Log Format
+category: features
+categories_url: features
+excerpt: Learn how to use terragrunt provider cache.
+tags: ["log"]
+order: 280
+nav_title: Documentation
+nav_title_link: /docs/
+---
+
+## Custom Log Format
+
+Using this `--terragrunt-log-custom-format <format>` flag you can specify which information you want to output.
+
+### Placeholders
+
+The format string consists of placeholders and text. Placeholders start with the `%` sign. The simplest example:
+
+```shell
+--terragrunt-log-custom-format "%time %level %msg"
+```
+
+Output:
+
+```shell
+10:09:19.809 debug Running command: tofu --version
+```
+
+The double sign `%%` displays the percent sign as plain text.
+
+```shell
+--terragrunt-log-custom-format "%time %level %%msg"
+```
+
+Output:
+
+```shell
+10:09:19.809 debug %msg
+```
+
+Placeholders have preset names:
+
+* `%time` - Current time.
+
+* `%interval` - Seconds has passed since Terragrunt started.
+
+* `%level` - Log level.
+
+* `%prefix` - Path to working directory.
+
+* `%tfpath` - Path to TF executable file.
+
+* `%msg` - Log message.
+
+Any other text is considered as plain text, for example:
+
+```shell
+--terragrunt-log-custom-format "time=%time level=%level message=%msg"
+```
+
+Output:
+
+```shell
+time=00:10:44.716 level=debug message=Running command: tofu --version
+```
+
+A placeholder is just a value, to format this value you need to pass options to the placeholder. It has the following syntax:
+
+`%placeholder-name(option-name=option-value, option-name=option-value,...)`
+
+```shell
+--terragrunt-log-custom-format "%time(format='Y-m-d H:i:sv') %level(format=short,case=upper) %msg"
+```
+
+Output:
+
+```shell
+2024-11-12 11:52:20.214 DEB Running command: tofu --version
+```
+
+Even if you don't pass options, the empty brackets are added implicitly. Thus `%time` equals `%time()`. If you need to add brackets as plain text after a placeholder with no options and without space, you need to explicitly specify empty brackets first, otherwise, they will be treated as invalid options.
```

Comment:
In this text I wanted to clarify that it is necessary to explicitly specify empty parenthesis if it is necessary to specify parenthesis as plain text immediately after the placeholder. 

e.g.:

```shell
%time()(plain-text)
```
Output:

```shell
12:33:08.513(plain-text)
```

Because if it's specified like this

```shell
%time(plain-text)
```

it returns an error, invalid option name `plain-text`

Reply:
Agree with you. Updated docs.